### DISCUSSION TITLE
Linux Fullscreen Scaling

### DISCUSSION THREAD

**Discussion by juliansko** [kwsch/PKHeX#3578] [Q&A]
Hi,
is there a way to use the Mono Version with Fullscreen and proper sizes at 200% Scaling?
Because right now I have to switch the scaling before using PKHeX.
Thanks in Advance!

**Comment by kwsch** [comment]
Nope, not even sure what would be needed for that. The program isn't meant to be scaled.

**Reply by juliansko** [reply]
![image](https://user-images.githubusercontent.com/57212021/187930478-e1881b5b-b329-468f-a735-7186215d3f49.png)
That's what I meant with scaling, is it possible to make the program scaling independent from Desktop (in this case Gnome) scaling?

**Reply by kwsch** [reply]
Per the readme, we don't use any variants of Linux for development or testing. Mono runtime isn't supported beyond the guarantee that we're not doing anything Windows specific.
